app/Exceptions/GitNotAvailable.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Exceptions;
+
+use Exception;
+use Throwable;
+
+class GitNotAvailable extends Exception
+{
+	public function __construct(
+		$code = 0,
+		Throwable $previous = null
+	) {
+		parent::__construct('../.git (and subdirectories) are not executable, check the permissions.', $code, $previous);
```

Comment:
Should this read `(and/or its subdirectories)`? I think it gets the message across either way though.